MapboxDirections/MBVoiceInstruction.swift

```diff
@@ -3,30 +3,43 @@ import Foundation
 @objc(MBVoiceInstruction)
 public class VoiceInstruction: NSObject, NSSecureCoding {
     
-    public let distanceAlongGeometry: CLLocationDirection
+    /**
+     The distance in meters along the current `RouteStep` to give the voice announcement.
+     */
+    public let distanceAlongStep: CLLocationDistance
 
-    public let textAnnouncement: String
     
-    public let SSMLAnnouncement: String
+    /**
+     A plain text representation of the voice instruction to give to the user.
+     */
+    public let text: String
+    
+    
+    /**
+     The voice instruction formatted with [SSML](https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/docs/speech-synthesis-markup-language-ssml-reference) (Speech Synthesis Markup Language).
+     
+     Note, when speaking an instruction with this property, the speed synthesis engine must support SSML.
+     */
+    public let ssml: String
```

Comment:
Name this property `ssmlText` to make its relationship to `text` more clear. Also give it an `@objc(SSMLText)` attribute.